contracts/LifCrowdsale.sol

```diff
@@ -11,6 +11,10 @@ contract LifCrowdsale is Ownable, Pausable {
   // The token being sold
   LifToken public token;
 
+  // start and end of the public presale
+  uint256 public startPublicPresaleBlock;
+  uint256 public endPublicPresaleBlock;
```

Comment:
actually maybe we should add a privatePresaleEndBlock... and other variables are whatever other variables the publicPresale uses, for example `publicPresaleRate`